quilt/test/test_build.py

```diff
@@ -62,3 +62,13 @@ def test_build_parquet(self):
             'Expected dataframes to have same # columns'
         del os.environ["QUILT_PACKAGE_FORMAT"]
         # TODO add more integrity checks, incl. negative test cases
+
+    def test_generate_buildfile(self):
+        mydir = os.path.dirname(__file__)
+        path = os.path.join(mydir, 'data')
+        buildfilepath = os.path.join(path, 'build.yml')
```

Comment:
You should use the current directory instead - it's a temporary directory that will get removed when the test is done.